dev/InfoBar/TestUI/InfoBar.cs

```diff
@@ -43,7 +42,8 @@ public bool Cancel
             get; set;
         }
     }
-    public class InfoBarEventArgs : EventArgs
+
+    public class CloseButtonClickEventArgs : EventArgs
     {
         public bool IsHandled
```

Comment:
>IsHandled [](start = 20, length = 9)

Generally we just call this Handled ([https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/events-and-routed-events-overview#the-handled-property](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/events-and-routed-events-overview#the-handled-property))

Reply:
Makes sense! Thanks for the link, should be fixed with the latest push.